apps/test/unit/code-studio/components/progress/ProgressDotTest.js

```diff
@@ -443,7 +443,8 @@ describe('ProgressDot component tests', () => {
         );
 
         const result = renderer.getRenderOutput();
-        expect(result.props.children[0].props.className).to.equal('');
+        expect(result.props.children[0].type).to.equal('div');
+        expect(result.props.children[0].props.className).to.equal(undefined);
```

Comment:
Don't think it makes a big difference one way or the other really, but `undefined` seems slightly more accurate.